### DISCUSSION TITLE
DNS settings to postal.example.com ?

### DISCUSSION THREAD

**Discussion by solracsf** [postalserver/postal#1571] [Help with using Postal] [answered]
I've just [installed Postal 2](https://docs.postalserver.io/install/installation), and when setting up a new domain, DNS entries are requested to point to `postal.example.com`.

![image](https://user-images.githubusercontent.com/12234510/131218465-e6019580-a15e-4cf7-93bb-47c7c06c4059.png)

How to fix this?

**Comment by solracsf** [comment]
After `postal stop && postal start` (a `restart` command would be great) it works now.

**Comment by raf-rodrigues** [comment, marked as answer]
This correction you will make in the postal configuration file.

_/opt/postal/postal.yml_


`dns:
  mx_records:
    - mx.postal.example.com
  smtp_server_hostname: postal.example.com
  spf_include: spf.postal.example.com
  return_path: rp.postal.example.com
  route_domain: routes.postal.example.com
  track_domain: track.postal.example.com`

Change to your Postal server address.


To apply the settings

**postal stop**

**postal start**
